kuma/authkeys/views.py

```diff
@@ -12,7 +11,7 @@
 ITEMS_PER_PAGE = 15
 
 
-@login_required
+@permission_required('authkeys.add_key', raise_exception=True)
```

Comment:
The `permission_required` decorator doesn't check it, but also an anonymous user would not have those permissions. But I think I would add `login_required` b/c a login redirect is nicer than a 403.